app/service/planning_svc.py

```diff
@@ -104,6 +107,18 @@ def __init__(self):
                 link['command'] = self.encode_string(decoded_test)
         return links
 
+    async def _do_enforcements(self, ability_requirements, operation, link, combo):
+        for requirements_info in ability_requirements:
+            # list of facts that have a relationship
```

Comment:
remove the 2 comments in this function. they are repetitive: if a programmer can't figure out what these lines do, these comments are unlikely to help.